docs/deployment/deployK8s-AWS.md

```diff
@@ -5,343 +5,158 @@ title: Prepare AWS EKS for Multi-Region Deployments
-
-      * Under the **Select a local VPC to peer with** section, enter the `VpcId` value for the `us-west-2` VPC.
-
-      * In the **Select another VPC to peer with** list, select **My account** --> **Another region** --> **Canada Central** (ca-central-1).
-
-      * Under the **VPC (Accepter)** section, enter the `VpcId` value for the `ca-central-1` region.
+1. Create VPCs.
 
-      * Click **Create Peering Connection** and when you receive a confirmation message, click **OK** to continue.
+    * Sign on to the AWS console and navigate to the **VPC** service.
 
-      * At the top right of the page, change the region to **Canada Central**.
+    * Toggle to the `eu-west-1` region.
 
-      * Select **Peering Connections**.
+    * Select **Your VPCs** (under Virtual Private Cloud) and click **Create VPC**
 
-         > The peering connection status for `us-west-2` will show as `Pending Acceptance`.
+    * Add a name tag, such as `demo-vpc-eu-west-1`
 
-      * Select the `ca-central-1` connection, click the **Actions** list, and select **Accept Request** and confirm when prompted.
+    * Add a IPv4 CIDR, such as `10.0.0.0/16`
 
-         > The VPC peering connection status should now show as `Active`.
+    * Click **Create VPC**.
 
-5. Get the subnet information for each cluster node.
+    > Make note of the `VpcId` and `IPv4 CIDR` values for the `eu-west-1` and `us-east-1` VPCs for use in subsequent steps.
 
-      Each cluster node uses a different subnet, so there are three subnets assigned to each VPC. The information displayed contains the subnet ID for each subnet. Use the subnet IDs in the subsequent step to get the associated routing tables.
+    * Repeat this step in `us-east-1` region.
 
-      * At the top right of the page, change the region to **Oregon**.
+2.  Create the transit gateway for each region your deployment is being hosted on. Toggle to the `eu-west-1` region.
 
-      * Go to the **EC2** service, and select **Instances**. Apply a filter, if needed, to find your nodes for the cluster.
+    * Navigate to the **Transit gateways** section and click **Create transit gateway**.
 
-      * Select each node and record the **Subnet ID** for use in a subsequent step.
+    * Add a name tag such as `demo-tgw-eu-west-1`.
 
-      * At the top right of the page, change the region to **Canada Central**, and repeat these steps to find and record the subnet IDs for this VPC.
+    * Add a unique Amazon side Autonomous System Number for each region (ex. 64512 or 64513).
 
-6. Get the routing table associated with the subnets for each VPC.
+    * Disable both the `Default route table association` and `Default route table propagation`.
+    
+    > Make Note if you Enable the options above the default association route table and propagation route table will be created. This may not suit your more complex routing needs, so below you can see how to manually set the associations/propagation route tables.
```

Comment:
**_Note: If you enable the options above,_** the default association route table and propagation route table will be created. This **_action_** may not **_suit more_** complex routing **_needs; see below for details on how to_** manually set the associations/propagation route tables.